drivers/wireless/gs2200m.c

```diff
@@ -2018,7 +2018,10 @@ static enum pkt_type_e gs2200m_send_bulk(FAR struct gs2200m_dev_s *dev,
     }
   else
     {
-      wlinfo("** addr=%s port=%d\n", inet_ntoa(msg->addr.sin_addr),
+      char inetaddr[INET_ADDRSTRLEN + 2];
```

Comment:
why +2 is needed?

Reply:
No clue actually, came from inetntoa static allocation.

Reply:
Here is the definition:
```
/* Sizes of addresses (per OpenGroup.org) */

#define INET_ADDRSTRLEN       16   /* nnn.nnn.nnn.nnn */
#define INET6_ADDRSTRLEN      46   /* xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx */
```
it's safe to define char inetaddr[INET_ADDRSTRLEN].

Reply:
Well I would add +1 for the zero terminator, question is why it used to be +2?

Reply:
No, nnn.nnn.nnn.nnn has 15(4*4 - 1) char. 16bytes is enough to contain nnn.nnn.nnn.nnn plus \0

Reply:
Yes. No +1 or +2 is needed

Reply:
> why +2 is needed?

This is why it is better not to put hard-coded constant literals in the code, unless there is a comment that explains the rationale behind that number.